src/widget/weffectbuttonparameter.cpp

```diff
@@ -26,4 +26,5 @@ void WEffectButtonParameter::setup(const QDomNode& node, const SkinContext& cont
         SKIN_WARNING(node, context)
                 << "EffectButtonParameter node could not attach to effect parameter";
     }
+    this->setFocusPolicy(Qt::NoFocus);
```

Comment:
this-> is redundant and should be removed.

Reply:
because this is a wpushbutton actually.
so the same applies to the weffectparameterknob.
will fix it